tensorflow/tools/dist_test/scripts/k8s_tensorflow.py

```diff
@@ -127,33 +151,51 @@
     """apiVersion: v1
 kind: Service
 metadata:
-  name: tf-ps{param_server_id}
+  name: {cluster_name}-tf-ps{param_server_id}
   labels:
+    tf-component: "parameter-server"
     tf-ps: "{param_server_id}"
+    ts-cluster-name: "{cluster_name}"
+    creator: "{creator}"
 spec:
   ports:
   - port: {port}
+    name: tensorflow
   selector:
     tf-ps: "{param_server_id}"
+    ts-cluster-name: "{cluster_name}"
 """)
 PARAM_LB_SVC = ("""apiVersion: v1
 kind: Service
 metadata:
-  name: tf-ps{param_server_id}
+  name: {cluster_name}-tf-ps{param_server_id}
   labels:
+    tf-component: "parameter-server"
     tf-ps: "{param_server_id}"
+    ts-cluster-name: "{cluster_name}"
+    creator: "{creator}"
 spec:
   type: LoadBalancer
   ports:
   - port: {port}
+    name: tensorflow
   selector:
     tf-ps: "{param_server_id}"
+    ts-cluster-name: "{cluster_name}"
 """)
 
 
 def main():
   """Do arg parsing."""
   parser = argparse.ArgumentParser()
+  parser.add_argument('--cluster_name',
+                      type=str,
+                      default=DEFAULT_CLUSTER_NAME,
```

Comment:
Same as below. Can you please make it backward compatible? Let the default be an empty string, which GenerateConfig() can ignore, which leads to the same naming pattern as before. Thanks!